browser/js/common/directives/navbar/navbar.html

```diff
@@ -106,9 +98,9 @@
 	<div ng-class='class'>
 		<div class='dropdown-lists'>
 
-			<div class='dropdown'>
-				<div class='dropdown-title' ng-click='goToSection(".overview", "home")'>What is Kinetic</div>
-				<div class='dropdown-content'>
+			<div class='dropdown' ng-init='whatIs = true'>
+				<div class='dropdown-title' ng-click="whatIs = !whatIs; getInvolved = true; resources = true; ourTeam = true">What is Kinetic</div>
+				<div ng-class="{hide: whatIs, show: !whatIs}">
```

Comment:
I don't know if this should be pulled out into the navbar.js file and if so I wasn't sure how to reference the variable since there was something going on with me having to call $digest since the DOM wasn't calling it I think.